build/pkgs/gmp/spkg-configure.m4

```diff
@@ -2,6 +2,26 @@ SAGE_SPKG_CONFIGURE([gmp], [
            sage_spkg_install_gmp=no
             AC_CHECK_HEADER(gmp.h, [], [sage_spkg_install_gmp=yes])
             AC_CHECK_HEADER(gmpxx.h, [], [sage_spkg_install_gmp=yes])
+            # Check that GMP >= 6.2.1; this test is copied from FLINT.
```

Comment:
Is this alignment correct. See line 2 and line 3.

Reply:
improved in 090a5af531f